tensorflow-core/tensorflow-core-api/src/gen/annotations/org/tensorflow/op/EstimatorOps.java

```diff
@@ -0,0 +1,607 @@
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ==============================================================================
+//
+// This class has been generated, DO NOT EDIT!
+//
+package org.tensorflow.op;
+
+import org.tensorflow.Operand;
+import org.tensorflow.op.estimator.BoostedTreesAggregateStats;
+import org.tensorflow.op.estimator.BoostedTreesBucketize;
+import org.tensorflow.op.estimator.BoostedTreesCalculateBestFeatureSplit;
+import org.tensorflow.op.estimator.BoostedTreesCalculateBestFeatureSplitV2;
+import org.tensorflow.op.estimator.BoostedTreesCalculateBestGainsPerFeature;
+import org.tensorflow.op.estimator.BoostedTreesCenterBias;
+import org.tensorflow.op.estimator.BoostedTreesCreateEnsemble;
+import org.tensorflow.op.estimator.BoostedTreesCreateQuantileStreamResource;
+import org.tensorflow.op.estimator.BoostedTreesDeserializeEnsemble;
+import org.tensorflow.op.estimator.BoostedTreesEnsembleResourceHandleOp;
+import org.tensorflow.op.estimator.BoostedTreesExampleDebugOutputs;
+import org.tensorflow.op.estimator.BoostedTreesFlushQuantileSummaries;
+import org.tensorflow.op.estimator.BoostedTreesGetEnsembleStates;
+import org.tensorflow.op.estimator.BoostedTreesMakeQuantileSummaries;
+import org.tensorflow.op.estimator.BoostedTreesMakeStatsSummary;
+import org.tensorflow.op.estimator.BoostedTreesPredict;
+import org.tensorflow.op.estimator.BoostedTreesQuantileStreamResourceAddSummaries;
+import org.tensorflow.op.estimator.BoostedTreesQuantileStreamResourceDeserialize;
+import org.tensorflow.op.estimator.BoostedTreesQuantileStreamResourceFlush;
+import org.tensorflow.op.estimator.BoostedTreesQuantileStreamResourceGetBucketBoundaries;
+import org.tensorflow.op.estimator.BoostedTreesQuantileStreamResourceHandleOp;
+import org.tensorflow.op.estimator.BoostedTreesSerializeEnsemble;
+import org.tensorflow.op.estimator.BoostedTreesSparseAggregateStats;
+import org.tensorflow.op.estimator.BoostedTreesSparseCalculateBestFeatureSplit;
+import org.tensorflow.op.estimator.BoostedTreesTrainingPredict;
+import org.tensorflow.op.estimator.BoostedTreesUpdateEnsemble;
+import org.tensorflow.op.estimator.BoostedTreesUpdateEnsembleV2;
+import org.tensorflow.op.estimator.IsBoostedTreesEnsembleInitialized;
+import org.tensorflow.op.estimator.IsBoostedTreesQuantileStreamResourceInitialized;
+import org.tensorflow.types.TFloat32;
+import org.tensorflow.types.TInt32;
+import org.tensorflow.types.TInt64;
+import org.tensorflow.types.TString;
+import org.tensorflow.types.family.TType;
+
+/**
+ * An API for building {@code estimator} operations as {@link Op Op}s
+ *
+ * @see {@link Ops}
+ */
+public final class EstimatorOps {
```

Comment:
Didn't estimators get removed in TF 2.16? Or is this a different meaning of estimator? It looks like it's about boosted trees and all that stuff got move out into tfdf.

Reply:
Yeah... we have different options:
    1. Preserve the operator and the Ops classes but marking them as deprecated (the generator used to support that and I think that is still the case)
    2. Leave the operator classes but not the Ops classes
    3. Discard all of them

Reply:
Yep, looks like everything under `estimator` in Python is deprecated: https://www.tensorflow.org/versions/r2.15/api_docs/python/tf/estimator

Since we haven't released anything officially yet, I suggest that we skip them entirely

Reply:
Sounds good.